cmd/kyma/alpha/enable/module/module.go

```diff
@@ -122,11 +122,25 @@ func (cmd *command) run(ctx context.Context, l *zap.SugaredLogger, moduleName st
 
 	kyma := types.NamespacedName{Name: cmd.opts.KymaName, Namespace: cmd.opts.Namespace}
 	moduleInteractor := module.NewInteractor(l, cmd.K8s, kyma, cmd.opts.Timeout, cmd.opts.Force)
-	modules, err := moduleInteractor.Get(ctx)
+	modules, kymaChannel, err := moduleInteractor.Get(ctx)
 	if err != nil {
 		return fmt.Errorf("failed to get modules: %w", err)
 	}
 
+	templates, err := moduleInteractor.GetAllModuleTemplatesOfModule(ctx, moduleName)
+	if err != nil {
+		return err
+	}
+	channelToLookup := cmd.opts.Channel
+	if channelToLookup == "" {
+		channelToLookup = kymaChannel
+	}
+	if !doesChannelExist(templates, channelToLookup) {
+		return errors.Errorf("The Channel [%s] does not exist for the ModuleTemplate [%s]\n"+
```

Comment:
```suggestion
		return fmt.Errorf("the channel [%s] does not exist for the ModuleTemplate [%s]\n"+
```
Nice error message! But try to keep it consistent with go errors and do not use line-breaks if possible